src/language/class-modifiers.md

```diff
@@ -0,0 +1,245 @@
+---
+title: Class modifiers
+description: Modifier keywords for class declarations to control external library access.
+---
+
+Class modifiers control how a class or a mixin can be subtyped in libraries
```

Comment:
except for `abstract`, right?

Reply:
I'm not sure... maybe the most clear explanation is to explicitly call out that `abstract` applies to all classes regardless of the library, and the rest of the modifiers control how a class or mixin can be subtyped.

Reply:
The `mixin` modifier applies to code within the same library too.